<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RosBotMainWindow</class>
 <widget class="QMainWindow" name="RosBotMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>590</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1022</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RosBotMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_16">
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_diag">
      <property name="title">
       <string>Diagnostic messages</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextBrowser" name="textBrowser_log">
         <property name="placeholderText">
          <string>Robot Teleop GUI Version 0.01</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_battery">
      <property name="title">
       <string>Battery levels:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="1" column="0">
        <widget class="QLabel" name="label_pwrRPi">
         <property name="text">
          <string>RPi:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="pBar_RPi">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_pwrMotor">
         <property name="text">
          <string>Power train:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="pBar_Motor">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs_teleop">
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Movement">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Movement</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_movement">
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout_speed">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_speed">
              <property name="title">
               <string>Speed override:</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_min">
                 <property name="text">
                  <string>Min</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="speedSlider">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="tracking">
                  <bool>false</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_max">
                 <property name="text">
                  <string>Max</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupBox_linear">
            <property name="title">
             <string>Linear movement:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_direction" columnstretch="0,0,0" columnminimumwidth="0,0,0">
             <item row="2" column="1">
              <widget class="QPushButton" name="Button_backward">
               <property name="text">
                <string>Backward</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="Button_forward">
               <property name="text">
                <string>Forward</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="Button_left">
               <property name="text">
                <string>Left</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="Button_right">
               <property name="text">
                <string>Right</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_angular">
            <property name="title">
             <string>Angular movement:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_rotate">
             <item row="0" column="0">
              <widget class="QPushButton" name="Button_zminus">
               <property name="text">
                <string>Rotate Z+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="Button_zplus">
               <property name="text">
                <string>Rotate Z-</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_gestures">
       <attribute name="title">
        <string>Gestures</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox_wave">
          <property name="title">
           <string>Wave with leg:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="Button_waveFLeft">
             <property name="text">
              <string>Front Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_waveFRight">
             <property name="text">
              <string>Front Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_waveRLeft">
             <property name="text">
              <string>Rear Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_waveRRight">
             <property name="text">
              <string>Rear Right</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox_lean">
          <property name="title">
           <string>Lean direction:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="Button_leanLeft">
             <property name="text">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_leanRight">
             <property name="text">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_leanForward">
             <property name="text">
              <string>Forward</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Button_leanBackward">
             <property name="text">
              <string>Backward</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_scared">
          <property name="title">
           <string>Scared</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="Button_animate">
             <property name="text">
              <string>Animate</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Layout_shakes">
             <item>
              <widget class="QLabel" name="label_shakes">
               <property name="text">
                <string>Shakes:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_shakes"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Layout_Beeps">
             <item>
              <widget class="QLabel" name="beeps_label">
               <property name="text">
                <string>Beeps:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="beeps_spinBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_chirp">
          <property name="title">
           <string>Chirp</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="Button_play">
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="beepsLayout">
             <item>
              <widget class="QLabel" name="label_beeps">
               <property name="text">
                <string>Beeps:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_beeps"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_configuration"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Contents"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionServo_configuration"/>
    <addaction name="actionJoint_configuration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_configuration"/>
   <addaction name="actionSave_configuration"/>
   <addaction name="separator"/>
   <addaction name="actionJoint_configuration"/>
   <addaction name="actionServo_configuration"/>
   <addaction name="separator"/>
   <addaction name="action_Contents"/>
   <addaction name="action_About"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Contents">
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_configuration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open configuration...</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save configuration...</string>
   </property>
  </action>
  <action name="actionServo_configuration">
   <property name="icon">
    <iconset theme="emblem-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Servo configuration...</string>
   </property>
  </action>
  <action name="actionJoint_configuration">
   <property name="icon">
    <iconset theme="emblem-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Joint configuration...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
